common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -356,6 +353,17 @@ public void onReceive(Context context, Intent intent) {
         }
     }
 
+    /**
+     * Cancel any current interactive requests.
+     * @param broadcastManager the broadcast manager to use to cancel requests.
+     */
+    public static void cancelInteractiveRequest(LocalBroadcastManager broadcastManager) {
```

Comment:
nit: `NonNull` and `final`

Reply:
OK.  In case we need it.  Otherwise we shouldn't ship this.  I'm going to fix this and make it a draft.